cfme/web_ui/__init__.py

```diff
@@ -1190,10 +1191,34 @@ class Form(Region):
     pretty_attrs = ['fields']
 
     def __init__(self, fields=None, identifying_loc=None):
-        self.locators = dict((key, value) for key, value in fields)
+        self.metadata = {}
+        self.locators = {}
+        for field in fields:
```

Comment:
Could be, but I made it spit the ValueError since now people can start making mistakes when dealing with multi-lengthed tuples ...